packages/api-client/src/components/ApiClient/Response/ResponseBody.vue

```diff
@@ -16,14 +16,14 @@ const mediaType = computed(() => {
   const headers = normalizeHeaders(props.response?.headers)
 
   // Get the content-type header
-  const contentTypeHeader = headers['content-type']
+  const contentTypeHeader = `${headers['content-type']}`
```

Comment:
I'm coercing this to a string now so it gets typed as `string` instead of `any`.